src/app/find-account/panel/SendCodeForm.tsx

```diff
@@ -126,7 +126,9 @@ const SendCodeForm = ({
                   ),
                 }}
               />
-              {errors.code && <Typography>{errors.code.message}</Typography>}
+              {(errors.code && (
+                <Typography variant="Caption">{errors.code.message}</Typography>
+              )) || <Typography variant="Caption">&nbsp;</Typography>}
```

Comment:
이부분 코드가 잘 이해되지 않는데 errors.code가 없는 경우에 `&nbsp;` 를 넣기 위한 부분인가요?
만약에 제가 제대로 이해한거라면 아래처럼 쓰면 좀 더 이해하기 쉬울 것 같아요..!
```suggestion
              <Typography variant="Caption">
                {errors.code ? errors.code.message : ' '}
              </Typography>
```

Reply:
아..! 이렇게 수정할게요! 회원가입 페이지에 적용되어있던 것들 그대로 가져오느라 이렇게 바꿀 생각은 못했네요😅